recipes/nltk_data/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+NTLK_DATA=${PREFIX}/lib/nltk_data
+mkdir -vp ${NTLK_DATA}
```

Comment:
Small comment. What about putting it in `share` or similar?

Reply:
What I would have chosen, but `nltk` doesn't look there unfortunately. Only `$PREFIX/nltk_data` or `$PREFIX/lib/nltk_data`: https://github.com/nltk/nltk/blob/2398bbae371b5ca8fce01cdd3e2eada99dea82d7/nltk/data.py#L103-L110

Reply:
Actually I'll send a PR to nltk to support `share` too; it looks in `/usr/share`, so stupid not to do it in `sys.prefix` too. But that'll have to wait for next nltk release, of course.

Reply:
Yeah I like that more but the only path that is searched under `sys.prefix` is `lib`: https://github.com/nltk/nltk/blob/2398bbae371b5ca8fce01cdd3e2eada99dea82d7/nltk/data.py#L110

Reply:
Ok. It's not a blocking issue. Just wondering since that's typically where data like this lives. Can be an issue on the feedstock. 😉

Reply:
https://github.com/nltk/nltk/pull/1904, but IMO until that's merged and released we should put it in `lib/nltk_data` rather than just root `nltk_data`.

Reply:
Fine with me. :)